docs/core/runtime-config/envvars.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Runtime configuration with environment variables"
+description: "Use environment variables to configure the runtime behavior of .NET Core applications."
+ms.date: "06/19/2019"
+author: "rpetrusha"
+ms.author: "ronpet"
+---
+# Runtime configuration with environment variables
+
+.NET Core supports a number of environment variables that you can use to configure your applications. 
+
+The following sections list selected supported environment variables by category. For a complete list of supported, as well as unsupported and internal environment variables, see [Host Configuration Knobs](https://github.com/dotnet/coreclr/blob/a28b25aacdcd2adb0fdfa70bd869f53ba6565976/Documentation/project-docs/clr-configuration-knobs.md).
```

Comment:
You should confirm with each of feature team that they are comfortable officially supporting each of these variables. I have commented on the obvious ones that we should not be documenting, but there is likely a bunch more.

A simple rule should be: All officially supported variables should have tests. If we do not have a test for it, we should not be declaring it as officially supported.

Reply:
Should this point to master?

```suggestion
The following sections list selected supported environment variables by category. For a complete list of supported, as well as unsupported and internal environment variables, see [Host Configuration Knobs](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/clr-configuration-knobs.md).
```